documentation/source/simulator_support.rst

```diff
@@ -16,6 +16,24 @@ Accessing bits of a vector doesn't work:
 See ``access_single_bit`` test in :file:`examples/functionality/tests/test_discovery.py`.
 
 
+Verilator
+---------
+
+cocotb supports Verilator 4.020 and above.
+Verilator converts Verilog code to C++ code that is compiled.
+It does not support VHDL.
+One major limitation compared to standard Verilog simulators is that it does not support delays assignments.
+
+To run cocotb with Verilator, you need `verilator` in your path or it can be discoverd by pkg-config.
+
+Finally, cocotb currently generates a Verilator toplevel C++ simulation loop which is timed at the highest precision.
+If your design's clocks vary in precision, the performance of the simulation can be improved in the same order of magnitude by adjusting the precision in the Makefile, e.g.,
+
+.. code-block:: makefile
+
+    PRECISION=6 # Set precision to 10^-6s
```

Comment:
Reflect the naming change here. Also, wouldn't it have been "-6" (https://github.com/chipsalliance/verilator/blob/master/include/verilated.h#L709)?